content/en/guide/v10/server-side-rendering.md

```diff
@@ -7,8 +7,6 @@ description: 'Render your Preact application on the server to show content to us
 
 Server-Side Rendering (often abbreviated as "SSR") allows you to render your application to an HTML string that can be sent to the client to improve load time. Outside of that there are other scenarios, like testing, where SSR proves really useful.
 
-> Note: SSR is automatically enabled with `preact-cli` :tada:
-
```

Comment:
Eventually I'd like to write up a page detailing SSR w/ the Vite preset (especially as I crammed _a lot_ of info into the preset's readme) but... not today.